testing/resources/selective_script_hooks_repo/hooks.yaml

```diff
@@ -0,0 +1,5 @@
+-   id: bash_hook
+    name: Bash hook
+    entry: bin/hook.sh
+    language: script
+    files: '^subdir/'
```

Comment:
you can probably just reuse the bash hook and modify the configuration to have this added on -- that said, I don't think this hook repo actually adds any coverage.  You really only need a test which cds and uses relative paths